docs/touch-id.md

```diff
@@ -0,0 +1,15 @@
+## Touch ID
+
+Appium has the capability to simulate Touch ID (https://support.apple.com/en-ca/HT201371) on iOS Simulators.
+
+### Support
+* TouchID simulation is only supported in iOS Simulators. It is not possible to simulate touchId on real devices.
+* Not all iOS devices have touchId so your tests should handle the case where touchId is not supported
+
+### Configuration
+* To use touchId, Terminal must be added to the accessibility preferences on your Mac (navigate to `System Preferences > Security & Privacy > Accessibility` and under `Allow the apps below to control your computer` add `Terminal`)
```

Comment:
strictly speaking, this isn't `Terminal`. It's whatever app you use to launch Appium from. So it might be Appium Desktop (if you're running that server). Or for me it'd be `iTerm`.

I also think it'd be good to talk about _why_ we require this. I.e.:

> The only way Appium can enable enrollment and toggling of Touch ID is to use system-level accessibility APIs to simulate mouse clicks on the Simulator menus via AppleScript. For this reason this feature requires that you give Appium's running context (your terminal application, for example) access to these accessibility APIs.